common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -338,7 +339,8 @@ object Activation extends DefaultJsonProtocol {
       "kind",
       "conductor",
       "memory",
-      "causedBy")
+      "causedBy",
+      "size")
```

Comment:
Should we also update the [documentation](https://github.com/apache/openwhisk/blob/master/docs/metrics.md#supported-events) with a short description of the new field? E.g.:  
```
...
memory - maximum memory allowed for action container
size - action response size in bytes
```
and 
```
{"body":{"statusCode":0,"duration":3,"name":"whisk.system/invokerHealthTestAction0","waitTime":583915671,"conductor":false,"kind":"nodejs:6","initTime":0,"memory": 256,"size": 128, "causedBy": false},"eventType":"Activation","source":"invoker0","subject":"whisk.system","timestamp":1524476122676,"userId":"d0888ad5-5a92-435e-888a-d55a92935e54","namespace":"whisk.system"}
```

Reply:
Updated the docs to document the `size` metrics